advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -171,6 +171,7 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
 }
 
 private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulnerability {
+    // FIXME: Clarify which entry is relevant and pick the right one accordingly.
```

Comment:
So far, we basically only use `TODO` in the code base (modulo a few places that you introduced). As I don't see a relevant difference between `TODO` and `FIXME`, can we stich to `TODO` so we'd only need to search for one term instead of two?